src/pages/Task.tsx

```diff
@@ -114,6 +114,7 @@ const TaskCore: React.FC = () => {
   const fileCount = get(data, "taskFiles.fileCount");
   const logLinks = get(task, "logs");
   const isPerfPluginEnabled = get(task, "isPerfPluginEnabled");
+  const canModifyAnnotation = get(task, "canModifyAnnotation");
```

Comment:
this is better written as `task?.canModifyAnnotation ?? false` so we can get closer to removing the `get` module

Reply:
Why do we not like the get module?

Reply:
[optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) fulfills our use case for `get` and doesn't require an additional installation